src/controllers/device.controller.ts

```diff
@@ -0,0 +1,22 @@
+import type {
+    Handler,
+    Request,
+    Response,
+  } from 'express';
+import { deviceService } from '../services';
+
+const addDevice: Handler = (
+    req: Request,
+    res: Response,
+): void => {
+    deviceService.addDevice(req.body)
+    .then((data) => res.json(data))
```

Comment:
Let's also return a status code of [HTTP `201 Created`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/201):

> The HTTP `201 Created` success status response code indicates that the request has succeeded and has led to the creation of a resource. The new resource is effectively created before this response is sent back and the new resource is returned in the body of the message, its location being either the URL of the request, or the content of the `Location` header.
> 
> The common use case of this status code is as the result of a `POST` request.

(I know we're not doing that for creating notifications yet, but we should be!)

Reply:
I would still like for us to do this!

Reply:
This is a To-do